### DISCUSSION TITLE
Merged PRs not coming in leaderboard for a repository.

### DISCUSSION THREAD

**Discussion by meetarora10** [sanjay-kv/Open-source-Practice#1039] [Q&A] [answered]
Just a few hours ago my points were 1700+ and now dropped to 1240 and I have noticed that in my Stats all my merged PR's from Devlabs repository are not there which has reduced my points .
Kindly have a look @sanjay-kv

**Comment by codeftsaikat** [comment]
Here are a few steps that might help restore your points

Check PR Status: Confirm that the PRs are still merged and haven’t been reverted or closed. Sometimes, repository maintainers will modify branches, which can affect contribution records.
Wait for a Sync Update: Occasionally, GitHub’s contribution stats don’t immediately reflect changes. Sometimes, waiting a few hours or up to a day can allow the system to update.
Reach Out to GitHub Support: If the points don’t restore or the PRs still aren’t showing, contacting GitHub Support could help. They might be able to refresh your stats manually or clarify any repository changes affecting your contributions.

**Comment by Randil-Hasanga** [comment]
If your merged pull requests aren’t showing up on the leaderboard, it might be due to a delay in leaderboard updates, or there could be a syncing issue with the repository. Check if the leaderboard refreshes periodically or if there’s a manual update required. If the issue persists, consider reaching out to the repository maintainer for assistance.

**Comment by aslams2020** [comment, marked as answer]
This Discussion belongs to last GSSoC program
